policy/modules/services/dovecot.if

```diff
@@ -61,6 +61,85 @@ interface(`dovecot_domtrans_deliver',`
 	domtrans_pattern($1, dovecot_deliver_exec_t, dovecot_deliver_t)
 ')
 
+########################################
+## <summary>
+##	Read and write dovecot unix
+##	stream sockets.
+## </summary>
+## <param name="domain">
+##	<summary>
+##	Domain allowed access.
+##	</summary>
+## </param>
+#
+interface(`dovecot_rw_stream_sockets',`
+	gen_require(`
+		type dovecot_t;
+	')
+
+	allow $1 dovecot_t:unix_stream_socket { connectto rw_stream_socket_perms };
+')
+
+########################################
+## <summary>
+##	Read and write dovecot runtime
+##	sock files.
+## </summary>
+## <param name="domain">
+##	<summary>
+##	Domain allowed access.
+##	</summary>
+## </param>
+#
+interface(`dovecot_rw_runtime_sock_files',`
+	gen_require(`
+		type dovecot_runtime_t;
+	')
+
+    rw_sock_files_pattern($1, dovecot_runtime_t, dovecot_runtime_t)
+')
```

Comment:
I would probably add a files_search_pids($1) here (if it is still called that)

Reply:
`files_search_pids` was renamed to `files_search_runtime`, but yes I will add that here.

Reply:
This should probably moved into the stream socket connection interface. This is pretty much stream_connect_pattern() but with a read on the `sock_file`.

Reply:
OK.